cmd/avsc2avdl/testdata/enumwarning.txt

```diff
@@ -0,0 +1,28 @@
+avsc2avdl issue2866.avsc
```

Comment:
This is awesome :heart:

Reply:
i added another one for the record defaults.
i also changed it so that the error message pinpoints the position in the output file instead of just naming the input file, so it should be easier to find and fix these issues.